vignettes/logistic-metadata-example.Rmd

```diff
@@ -521,7 +526,11 @@ EFIstandards::forecast_validator(my_eml)
 We are now ready to write out a valid EML document: 
 
 ```{r}
-write_eml(my_eml, "forecast-eml.xml")
+## file name
+emlname <- system.file("extdata", "forecast-eml.xml", package="EFIstandards")
```

Comment:
If I recall, we weren't writing the files to extdata automatically because we were getting a lot of trivial PR diffs (which were especially annoying on the binary netcdf files) and elected instead to just have the vignette build in place and to remember to update extdata manually when needed

Reply:
that makes sense but should document somewhere (e.g. I left a commented block up at line 14)